apps/webpack.config.js

```diff
@@ -136,6 +136,7 @@ const nodePolyfillConfig = {
       stream: require.resolve('stream-browserify'),
       timers: require.resolve('timers-browserify'),
       crypto: require.resolve('crypto-browserify'),
+      vm: require.resolve('vm-browserify'),
```

Comment:
This fixes a `yarn build` webpack warning I /thought/ I had generated on my branch as a result of moving deps, but I'm actually seeing it on other branches too, I just hadn't noticed. Anyway, this fixes it.